Assets/AltTester/Runtime/AltDriver/Commands/FindObjects/AltWaitForComponentProperty.cs

```diff
@@ -29,16 +31,18 @@ public class AltWaitForComponentProperty<T> : AltBaseFindObjects
         string propertyName;
         T propertyValue;
         string assembly;
+        bool getPropertyAsString;
         double timeout;
         double interval;
 
-        public AltWaitForComponentProperty(IDriverCommunication commHandler, string componentName, string propertyName, T propertyValue, string assemblyName, double timeout, double interval, AltObject altObject) : base(commHandler)
+        public AltWaitForComponentProperty(IDriverCommunication commHandler, string componentName, string propertyName, T propertyValue, string assemblyName, bool getPropertyAsString, double timeout, double interval, AltObject altObject) : base(commHandler)
```

Comment:
We should add the new parameter as the last one, so we don't mess up the calls for people that are using the method without explicit parameters - in these cases, their `timeout` param would be interpreted as the `getPropertyAsString`. This applies everywhere I guess

Reply:
Done. Related to the docs part, the changes are made here https://github.com/alttester/AltTester-Unity-SDK/pull/1628